compatibility/core/pom.xml

```diff
@@ -1,15 +1,17 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>org.mule.transports</groupId>
-        <artifactId>mule-transports</artifactId>
+        <groupId>org.mule.compatibility</groupId>
+        <artifactId>mule-compatibility</artifactId>
         <version>4.0-SNAPSHOT</version>
     </parent>
-    <artifactId>mule-transport-core</artifactId>
+    <groupId>org.mule</groupId>
+    <artifactId>mule-compatibility-core</artifactId>
     <packaging>jar</packaging>
     <name>Transports Core Infrastructure</name>
```

Comment:
This should be "Compatiblity Core Infrastructure". Similar change on the description tag